### ISSUE TITLE
failed to load module for ssr cause of removed buttons

### ISSUE THREAD

**Issue by WolfspiritM** [themesberg/flowbite-svelte#126]
Thanks for taking the time to fill out this bug report!

**Describe the bug**
Since removing the buttons with version 0.19.13 flowbite-svelte doesn't work anymore. Whenever I import anything I receive:
```
failed to load module for ssr: ./buttons/ColorShadowButton.svelte
Error: failed to load module for ssr: ./buttons/ColorShadowButton.svelte
    at instantiateModule (/workspaces/sveltetest/node_modules/vite/dist/node/chunks/dep-80fe9c6b.js:50206:15)
```
Moving back to 0.19.12 works. The reason is that these buttons are still exported from index.ts

**To Reproduce**
Use 0.19.13 and import e.g. a Card.

**Code samples**
```svelte
<script>
	import { Alert } from 'flowbite-svelte';
</script>
<Alert id="alert-red" color="red" closeBtn>
	<span slot="content"> Test </span>
</Alert>

```

**Expected behavior**
No error

**Comment by shinokada** [comment]
Sorry for the problem.
We deprecated ColorShadowButton. 
Button component has a new API and please use [this example](https://flowbite-svelte.com/buttons#Colored_shadows).

```
<Button coloredShadow="blue">Blue</Button>
<Button coloredShadow="green">Green</Button>
<Button coloredShadow="cyan">Cyan</Button>
<Button coloredShadow="teal">Teal</Button>
<Button coloredShadow="lime">Lime</Button>
<Button coloredShadow="red">Red</Button>
<Button coloredShadow="pink">Pink</Button>
<Button coloredShadow="purple">Purple</Button>
```

**Comment by WolfspiritM** [comment]
> Button component has a new API and please use [this example](https://flowbite-svelte.com/buttons#Colored_shadows).
I didn't use button at all. The problem was that when using something like ```import { Alert } from 'flowbite-svelte';``` with version 0.19.13 then it broke cause index.ts still exported the missing button files.

This is fixed with 0.20 so I'm going to close this issue :-) Thanks
